src/wpool_process_callbacks.erl

```diff
@@ -0,0 +1,76 @@
+-module(wpool_process_callbacks).
+
+-behaviour(gen_event).
+
+%% gen_event callbacks
+
+-export([ init/1
+        , handle_event/2
+        , handle_call/2
+        , handle_info/2
+        , code_change/3
+        , terminate/2]).
+
+-export([notify/3]).
+-type state() :: ordset:ordset(module()).
+
+-type event() :: on_init_start | on_new_worker | on_worker_dead.
```

Comment:
This module will define `-callbacks` for these 3 event types, right?

Reply:
Also, to preserve the style of other behaviors, I would call them `handle_worker_init`, `handle_worker_creation` and `handle_worker_death` (or similar `handle_<something>` names).